rdr_service/alembic/env.py

```diff
@@ -159,6 +159,7 @@ def run_migrations_online():
                 target_metadata=target_metadata.get(name),
                 include_schemas=True,
                 include_object=include_object_fn(name),
+                process_revision_directives=database.AutoHistoryRevisionGenerator.process_revision_directives
```

Comment:
Does this process every migration, or just the migrations that are greater than the current revision head?

Reply:
Sounds good.